### PR TITLE
Added measure bundle json to Measure

### PR DESCRIPTION
**Pull request by RohitKandimalla**
## MADiE PR Template

Jira Ticket: [MAT-5513](https://jira.cms.gov/browse/MAT-5513)
(Optional) Related Tickets:

### Summary

Adding measure bundle JSON to Measure so that we can save JSON and lock it down when a measure is versioned. 

### All Submissions
* [ ] This PR has the JIRA linked.
* [ ] Required tests are included.
* [ ] No extemporaneous files are included (i.e. Complied files or testing results).
* [ ] This PR is merging into the **correct branch**.
* [ ] All Documentation needed for this PR is Complete (or noted in a TODO or other Ticket).
* [ ] Any breaking changes or failing automations are noted by placing a comment on this PR.

### DevSecOps
If there is a question if this PR has a security or infrastructure impact, please contact the Security or DevOps engineer assigned to this project to discuss it further.

* [ ] This PR has NO significant security impact (i.e Changing auth methods, Adding a new user type, Adding a required but vulnerable package).

### Reviewers
By Approving this PR you are attesting to the following:

*  Code is maintainable and reusable, reuses existing code and infrastructure where appropriate, and accomplishes the task’s purpose.
*  The tests appropriately test the new code, including edge cases.
*  If you have any concerns they are brought up either to the developer assigned, security engineer, or leads.

### CODE REVIEW

File: src/main/java/gov/cms/madie/models/measure/Measure.java [hunk 1]
```diff
@@ -112,6 +112,7 @@ public class Measure extends ResourceAcl {
   @Singular("rav")
   private List<DefDescPair> riskAdjustments;
   private ProgramUseContext programUseContext;
+  private String measureBundleJson;
```
Comment (nmorasb) [inline src/main/java/gov/cms/madie/models/measure/Measure.java:115]: as mentioned in Slack, should we save this directly to the measure? As it will be a large field, we may want to store this in a separate collection and only fetch when actually needed for an operation.

**Review by nmorasb — APPROVED** [review]
